Move Changelog into docs

trevorgerhardt opened the pull request.

trevorgerhardt committed:
Move changelog into docs

trevorgerhardt committed:
Reduce boldness

trevorgerhardt committed:
Use the image component in the changelog

trevorgerhardt committed:
Move truncation

abyrd commented:
Discussed today, @ansoncfit plans to change Docusaurus settings to format dates, remove some blog-specific styling etc.

ansoncfit committed:
rename changelog files with release dates

ansoncfit committed:
show only latest post in sidebar, remove reading time

ansoncfit committed:
Reduce heading size

ansoncfit committed:
see below → read more below

Even though it's more verbose, I prefer this change if we will be truncating posts.

ansoncfit committed:
Merge pull request #48 from conveyal/changelog-suggestions

Changelog suggestions

ansoncfit reviewed (approved):
With my suggestions from #48 merged, this looks good to me.
